game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/DiceServerEditor.java

```diff
@@ -36,6 +37,19 @@
               "<html>Opens email registration page to register with MARTI dice-roller.<br>"
                   + "Needs to be done once before MARTI dice server can be used.</html>")
           .build();
+  private final JButton helpButton =
+      new JButtonBuilder("Help")
+          .actionListener(
+              () ->
+                  JOptionPane.showMessageDialog(
```

Comment:
There is a build for dialogs, it *might* help for setting the title. Something to consider.

Reply:
showMessageDialog actually has a method to specify title so just used that.